cidemipyme-gatsby/src/assets/css/cide_style.css

```diff
@@ -21,7 +21,12 @@
 }
 
 .home-section {
-  background-image: url("../images/building.jpg");
+  background-color:rgb(239, 245, 255);
+  background-image: url("../images/cide_fb_7.jpg");
```

Comment:
is not better to use absolute paths?

Reply:
gatsby generates the static files and routes, that is why I can't use an absolute path.